.github/install-codecov.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+#  Copyright 2021 PolyCrypt GmbH
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+sudo apt-get update && sudo apt-get install -y \
+	libcurl4-openssl-dev \
+	libelf-dev \
+	libdw-dev \
+	cmake \
+	gcc \
+	binutils-dev \
+	libiberty-dev \
+	coreutils
+
+wget https://github.com/SimonKagstrom/kcov/archive/master.tar.gz
+tar xzf master.tar.gz
+cd kcov-master
+mkdir build
+cd build
+cmake ..
+make -j $(nproc)
+make install DESTDIR=../../kcov-build
+cd ../..
+rm -rf kcov-master
```

Comment:
Why didn't you integrate this script into the workflow file as, for example, done [here](https://github.com/codecov/example-rust)?

Reply:
Done, all in the CI file now.